srml/contracts/src/exec.rs

```diff
@@ -673,6 +679,8 @@ where
 		self.ctx.overlay.get_rent_allowance(&self.ctx.self_account)
 			.unwrap_or(<BalanceOf<T>>::max_value()) // Must never be triggered actually
 	}
+
+	fn current_block(&self) -> &T::BlockNumber { &self.current_block }
```

Comment:
```suggestion
	fn current_block(&self) -> T::BlockNumber { self.current_block }
```

Reply:
`T::BlockNumber` is `Copy` so this should work and there's probably no need to return a reference.